gulpfile.js

```diff
@@ -0,0 +1,557 @@
+const fs = require("fs");
```

Comment:
FWIW, gulp seems to support .ts pretty easily: "For TypeScript, rename to gulpfile.ts and install the ts-node module." https://gulpjs.com/docs/en/getting-started/javascript-and-gulpfiles